Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -290,6 +290,8 @@ for this property.
 - (void)_isJITEnabled:(void(^)(BOOL))completionHandler WK_API_AVAILABLE(macos(10.14.4), ios(12.2));
 - (void)_removeDataDetectedLinks:(dispatch_block_t)completion WK_API_AVAILABLE(macos(10.14.4), ios(12.2));
 
+- (void)_didFillPasswordForUsername:(NSString *)username;
```

Comment:
This SPI needs availability clauses: `WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))`